configs/modules.config

```diff
@@ -17,17 +17,25 @@ process {
     }
 
     withName: "TORCH_TUNE" {
-        ext.prefix = { "${parsed_json}".tokenize('/')[-1].tokenize('.')[0] }
+        ext.prefix = { "${combination_key}".split(' ')[0].split('\\.')[0..-2].join("_") + "-" + "${combination_key}".split(' ')[2] }
         publishDir = [
-            path: { "${params.outdir}/${workflow.runName}_" + "${workflow.start}/".replaceAll('[-:]', '_').split('\\.')[0] + "/" + "${parsed_json}".tokenize('/')[-1].tokenize('.')[0] },
-            mode: params.publish_dir_mode,
-            overwrite: true
+            path: { "${params.outdir}/${workflow.runName}_" + "${workflow.start}/".replaceAll('[-:]', '_').split('\\.')[0] + "/" + "${combination_key}".split(' ')[0].split('\\.')[0..-2].join("_") + "-" + "${combination_key}".split(' ')[2] },
```

Comment:
this is very unreadable.... could you put a comment nearby to explain the expected output format?